### ISSUE TITLE
AssertionError with example project

### ISSUE THREAD

**Issue by jason-weise** [mcansh/remix-fastify#38]
Hi there

I am testing the `remix-fastify` solution here and notice that the examlpe throws an AssertionError as soon as you build and run `npm run start`

I see the example was updated 6 days ago, this is without any changes to the code at all.
Any ideas?

```
AssertionError [ERR_ASSERTION]: The first character of a path should be `/` or `*`
    at Router.on (Q:\example\node_modules\find-my-way\index.js:107:3)
    at Object.addNewRoute (Q:\example\node_modules\fastify\lib\route.js:271:16)
    at Object.route (Q:\example\node_modules\fastify\lib\route.js:203:19)
    at Object.prepareRoute (Q:\example\node_modules\fastify\lib\route.js:142:18)
    at Object._get [as get] (Q:\example\node_modules\fastify\fastify.js:244:34)
    at remixFastify (Q:\example\node_modules\@mcansh\remix-fastify\src\plugin.ts:71:13)
    at Plugin.exec (Q:\example\node_modules\avvio\plugin.js:130:19)
    at Boot.loadPlugin (Q:\example\node_modules\avvio\plugin.js:272:10)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  generatedMessage: false,
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '=='
}

```

**Comment by mcansh** [comment]
theres some commits that havent been released, try this example https://github.com/mcansh/remix-fastify/tree/1a2819f03e448a79e06e98f37f87c6e7fa8d243e/example

**Comment by jason-weise** [comment]
Hi @mcansh that seemed to work now using the tree you linked.
Do you know when that migth be released to npm?

**Comment by jason-weise** [comment]
Hi @mcansh , sorry I spoke too soon.
When I said it is working now, I had tested it on my MacBook Pro, on Windows, exactly the same error occurs.

**Comment by jason-weise** [comment]
It looks like the current repo might work on MacOS, just not Windows.

**Comment by mcansh** [comment]
continued discussion in discord https://discord.com/channels/770287896669978684/1026241146252435476

**Comment by mcansh** [comment]
fixed by #31 and released in [1.12.0](https://github.com/mcansh/remix-fastify/releases/tag/1.12.0)
